mobile/src/libs/components/sign-background/styles.ts

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  wrapper: {
+    backgroundColor: '#507CEB',
```

Comment:
Please use AppColor enum

Reply:
It's ok for background, it's too specific color